.github/workflows/integration-tests.yaml

```
@@ -38,11 +40,140 @@
+    runs-on: "ubuntu-24.04"
+    needs: build-migration-image
+    if: ${{ needs.build-migration-image.result == 'success' && needs.build-migration-image.outputs.any_changed == 'true' }}
+    steps:
+      - name: Restore migration image from cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            migration_image.env
+            migration_image.tar
+          key: migration-image-${{ runner.os }}-${{ github.sha }}
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)